services/context/repo.go

```diff
@@ -544,6 +551,7 @@ func RepoAssignment(ctx *Context) context.CancelFunc {
 	ctx.Data["Title"] = owner.Name + "/" + repo.Name
 	ctx.Data["Repository"] = repo
 	ctx.Data["Owner"] = ctx.Repo.Repository.Owner
+	ctx.Data["DefaultBranch"] = ctx.Repo.Repository.DefaultBranch
```

Comment:
It doesn't seem right to bloat the "global" ctx data.

Actually you don't need to assign this value, because in template I guess you could directly use something like `.Repository.DefaultBranch` , see line 552

Reply:
dd95d1bbaf5a9888c350450fb5ba1482f8ddaa09